src/main/kotlin/entity/ChattORESpec.kt

```diff
@@ -45,6 +47,10 @@ object ChattORESpec : ConfigSpec("") {
         val joinDiscord by optional("**<player> has joined the network**")
         val leaveDiscord by optional("**<player> has left the network**")
     }
+    object nicknamePresets : ConfigSpec() {
```

Comment:
I was thinking more like this:
```kotlin
val nicknamePresets by optional(
    mapOf(
        "pride" to pridePresets,
        "birdflop" to birdflopPresets,
        "players" to playerPresets
    )
)
```

This would allow us to create entirely new groups without modifying the Konf configuration.